src/widgets/settingspages/GeneralPage.cpp

```diff
@@ -297,6 +297,27 @@ void GeneralPage::initLayout(GeneralPageView &layout)
         false,
         "Specify how Chatterino will handle messages that exceed Twitch "
         "message limits");
+    layout.addDropdown<std::underlying_type<UsernameRightClickBehavior>::type>(
+        "Username right-click behavior",
+        {
+            "Reply with modifier",
+            "Mention with modifier",
+            "Always mention",
+            "Always reply",
+        },
+        s.usernameRightClickBehavior,
+        [](auto index) {
+            return index;
+        },
+        [](auto args) {
+            return static_cast<UsernameRightClickBehavior>(args.index);
+        },
+        false,
+        "Specify how Chatterino will handle right-clicking a username in "
+        "chat.");
+    addKeyboardModifierSetting(layout,
```

Comment:
This adds `None` as a possible modifier, I don't think it makes sense to have None as an option here.
If anything, the "Right-click behaviour" and "Modifier right-click behaviour" settings should have "None" as options instead

Reply:
I've removed the `None` option for the modifier in the GUI, it's still possible to write an invalid config manually. If modifier is `None` and a username is right-clicked, Chatterino will now log a critical message saying that the config is invalid and bail.